lib/rucio/core/permission/atlas.py

```diff
@@ -1042,3 +1045,35 @@ def perm_add_bad_pfns(issuer, kwargs):
     elif kwargs['state'] == str(BadPFNStatus.SUSPICIOUS):
         return True
     return _is_root(issuer)
+
+
+def perm_remove_did_from_followed(issuer, kwargs):
+    """
+    Checks if an account can remove did from followed table.
+
+    :param issuer: Account identifier which issues the command.
+    :param kwargs: List of arguments for the action.
+    :returns: True if account is allowed, otherwise False
+    """
+    return _is_root(issuer)\
+        or has_account_attribute(account=issuer, key='admin')\
+        or rucio.core.scope.is_scope_owner(scope=kwargs['scope'], account=issuer)\
+        or kwargs['scope'] == 'mock'
+
+
+def perm_remove_dids_from_followed(issuer, kwargs):
+    """
+    Checks if an account can bulk remove dids from followed table.
+
+    :param issuer: Account identifier which issues the command.
+    :param kwargs: List of arguments for the action.
+    :returns: True if account is allowed, otherwise False
+    """
+    if _is_root(issuer) or has_account_attribute(account=issuer, key='admin'):
+        return True
+    scopes = [did['scope'] for did in kwargs['dids']]
+    scopes = list(set(scopes))
+    for scope in scopes:
+        if not rucio.core.scope.is_scope_owner(scope, issuer):
```

Comment:
Same comment as above (remove the condition on the scope and add one on `kwargs['account] == issuer`